vlogsApp/vlogsApp/Controller/DetailScreenViewController.swift

```diff
@@ -23,29 +26,46 @@ class DetailScreenViewController: UIViewController {
     
     func configUI() {
         
+        titleLabel.text = blogTitle
```

Comment:
This single line can stay in configUI function where we populate the text , the styling of the label should be where we initialise the label